strongbox-web-core/src/main/java/org/carlspring/strongbox/security/vote/FilterAccessDecisionManager.java

```diff
@@ -1,30 +0,0 @@
-package org.carlspring.strongbox.security.vote;
```

Comment:
Generally, I did wrong that I removed this file in this branch since it's not strictly related with this task. I rolled it back. 

`FilterAccessDecisionManager `is not used currently, since @carlspring had some problems accessing repositories anonymously couple of days ago and I realized it was the main reason. I've created new ticket [SB-814](https://youtrack.carlspring.org/issue/SB-814) to determine its necessity.

From the first point of view `FilterAccessDecisionManager `is useless, since `CustomAccessDecisionVoter` (a thing responsible for what you pointed to:

> I suppose that it related not only with metods, SB-603 assumes URL access restriction too. 

) is also used by `MethodAccessDecisionManager` and URL access is checked there.